tools/debugging/README.md

```diff
@@ -57,3 +57,48 @@ $ eog ${number_of_test_case}.png
 
 And then, you can see elements and caps graph in a pipeline.
 
+
+### gst-debugger
+gst-debugger (a.k.a Gstreamer Debugger) toolset allows to introspect gst-pipeline remotely. It provides graphical client, and GStreamer's plugin.
+This guide is written on Ubuntu 16.04 X86_64 distribution.
+
+#### Build the source code
+```bash
+$ git clone https://github.com/GNOME/gst-debugger.git
+$ cd gst-debugger
+$ git checkout 0.90.0
+$ ./audogen.sh
```

Comment:
typo 'audogen' -> 'autogen'

Reply:
for './autogen.sh', need 'libprotobuf-c' package.
but, `sudo apt install libprotobuf-c` doesn't work.

Reply:
Typo is fixed.   If you need meed "checking for ****" statement while running autogen.sh script.  You have to install required package. 

* In my own Ubuntu: 
```bash
 dpkg -l | grep protobuf
ii  libmirprotobuf3:amd64                         0.26.3+16.04.20170605-0ubuntu1.1             amd64        Display server for Ubuntu - RPC definitions
ii  libprotobuf-c-dev                             1.2.1-1                                      amd64        Protocol Buffers C static library and headers (protobuf-c)
ii  libprotobuf-c1                                1.2.1-1                                      amd64        Protocol Buffers C shared library (protobuf-c)
ii  libprotobuf-dev:amd64                         2.6.1-1.3                                    amd64        protocol buffers C++ library (development files)
ii  libprotobuf-lite9v5:amd64                     2.6.1-1.3                                    amd64        protocol buffers C++ library (lite version)
ii  libprotobuf9v5:amd64                          2.6.1-1.3                                    amd64        protocol buffers C++ library
ii  protobuf-c-compiler                           1.2.1-1                                      amd64        Protocol Buffers C compiler (protobuf-c)
ii  protobuf-compiler                             2.6.1-1.3                                    amd64        compiler for protocol buffer definition files
ii  python-protobuf                               2.6.1-1.3                                    amd64        Python bindings for protocol buffers
```

Reply:
Note: 
> GStreamer's plugin. This guide is written on Ubuntu 16.04 X86_64 distribution.

Reply:
What I needed is 'libprotobuf-c-dev'.

Reply:
Additionally need to install two packages, 'protobuf-compiler' and 'libboost-all-dev'

Reply:
You describe  why packaging (e.g., .deb, .rpm) is important. :)